pkg/maps/ctmap/ctmap.go

```diff
@@ -248,58 +252,20 @@ func doGC6(m *bpf.Map, filter *GCFilter) int {
 
 		entry := entryMap.(*CtEntry)
 
-		// FIXME create a single function for doGC4 and doGC6
-
-		if filter.fType&GCFilterByTime != 0 &&
-			entry.lifetime < filter.Time {
-
-			del = true
-		}
-
-		if filter.fType&GCFilterByIDsToKeep != 0 &&
-			// In CT's entries, saddr is the packet's receiver,
-			// which means, is the destination container IP.
-			nextKey.saddr.IP().Equal(filter.IP) {
-
-			ruleCtx := policy.RuleContext{
-				SecID:          policy.NumericIdentity(entry.src_sec_id),
-				Port:           nextKey.sport,
-				Proto:          uint8(nextKey.nexthdr),
-				L7RedirectPort: entry.proxy_port,
-				IsRedirect:     entry.proxy_port != 0,
-			}
-
-			// Check if the src_sec_id of that entry is not allowed
-			// to talk with the destination container IP.
-			if _, ok := filter.IDsToKeep[ruleCtx]; !ok {
-				del = true
-			}
-		}
-
-		if filter.fType&GCFilterByID != 0 &&
-			// In CT's entries, saddr is the packet's receiver,
-			// which means, is the destination container IP.
-			nextKey.saddr.IP().Equal(filter.IP) {
-
-			ruleCtx := policy.RuleContext{
-				SecID:          policy.NumericIdentity(entry.src_sec_id),
-				Port:           nextKey.sport,
-				Proto:          uint8(nextKey.nexthdr),
-				L7RedirectPort: entry.proxy_port,
-				IsRedirect:     entry.proxy_port != 0,
-			}
+		// In CT's entries, `saddr` is the packet's receiver,
+		// which means, is the destination's IP.
```

Comment:
(I appreciate that this comment was there before, but while we're modifying this code...)

If I do some minor rephrasing of the above comment, it seems equivalent to:

```
In CT entries, the source address of the conntrack entry is the place that the packet is received, therefore it's the destination IP
```

However, that doesn't make sense to me. Is there some way we can better explain this?

Reply:
changed to 
```
		// In CT entries, the source address of the conntrack entry (`saddr`) is
		// the destination of the packet received, therefore it's the packet's 
		// destination IP
```